content/docs/5.1/develop/clients/java.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Java Client
+description: Interact with TiKV using Java.
+menu:
+    "5.1":
+        parent: TiKV Clients
+        weight: 1
+---
+
+This guide introduces how to interact with TiKV using [Java Client](https://github.com/tikv/client-java).
+
+{{< warning >}}
+TiKV Java Client is developed and released using Java8. The minimum supported version of TiKV is 2.0.0.
+{{< /warning >}}
```

Comment:
why wrapping it with a warning? I think a note is sufficient.

Reply:
sure